test/system/billing_expenses_test.rb

```diff
@@ -4,8 +4,12 @@ class BillingExpensesTest < ApplicationSystemTestCase
   def setup
     superadmin = create :user
     @volunteer = create :volunteer
-    assignment = create :assignment, volunteer: @volunteer
-    create :hour, volunteer: @volunteer, hourable: assignment
+    @assignment = create :assignment, volunteer: @volunteer
+    @hour1 = create :hour, volunteer: @volunteer, hourable: @assignment, hours: '2', minutes: '30'
+    category = create :group_offer_category
+    @group_offer = create :group_offer, volunteers: [@volunteer], group_offer_category: category
```

Comment:
The group offer factory calls the group offer category one, so:
`, group_offer_category: category` and the above line are not needed.